### DISCUSSION TITLE
Manually trigger an exit event

### DISCUSSION THREAD

**Discussion by 3elDU** [hajimehoshi/ebiten#2402] [Q&A] [answered]
I have a simple state machine.
I check, when the last state exits, I want to exit the program. But how should I do this programmatically?
Sorry for a noob question, and thanks in advance.

**Comment by hajimehoshi** [comment, marked as answer]
For Ebitengine v2.4, return your defined error at `Update`. `RunGame` will return the same error.

For Ebitengine v2.5 (not released yet), you can return `Termination` at `Update.`

**Reply by 3elDU** [reply]
Thx!
